pkg/crc/ssh/client.go

```diff
@@ -93,6 +93,10 @@ func (client *NativeClient) session() (*ssh.Session, error) {
 func (client *NativeClient) Run(command string) ([]byte, []byte, error) {
 	session, err := client.session()
 	if err != nil {
+		if client.conn != nil {
+			client.conn.Close()
+			client.conn = nil
+		}
```

Comment:
What is a good way to test this code path with a crc VM? When is this needed?

Reply:
@cfergeau Good way to test this code path with crc VM is during the pull secret update and manually reboot the machine, I think without this patch it will not work at all.

Reply:
I would put this in the commit log :)

Reply:
But is this what happens when configuring a proxy? The exact sequence of events which happen with 4.8 when configuring a proxy, and which you are fixing with this PR is very unclear for me